app/models/participant_outcome/npq.rb

```diff
@@ -21,6 +21,54 @@ def self.latest
     order(created_at: :desc).first
   end
 
+  def self.to_send_to_qualified_teachers_api
+    eligible_outcomes = for_completed_declarations
+      .not_sent_to_qualified_teachers_api
+      .where(id: latest_per_declaration.map(&:id))
+
+    eligible_outcomes.passed
+      .or(
+        eligible_outcomes
+          .not_passed
+          .where(participant_declaration_id: declarations_where_previous_outcome_passed_and_sent),
+      )
+  end
+
+  class << self
+    def latest_per_declaration
+      select("DISTINCT ON(participant_declaration_id) *")
+        .order(:participant_declaration_id, created_at: :desc)
+    end
+
+    def for_completed_declarations
+      joins(:participant_declaration)
+        .where("participant_declarations.declaration_type='completed'")
```

Comment:
suggestion: maybe we can use the `completed` scope on declarations? Although I think we might not need this anymore since we are performing the queries from the outcomes side now

Reply:
discussed in person: no longer needed 👍